src/ui/SimpleFilter/SimpleFilterValues.ts

```diff
@@ -0,0 +1,65 @@
+import 'styling/_SimpleFilter';
+import { IDoneBuildingQueryEventArgs, IQuerySuccessEventArgs } from '../../events/QueryEvents';
+import { ComponentOptions, IFieldOption } from '../Base/ComponentOptions';
+import * as _ from 'underscore';
+import { IGroupByRequest } from '../../rest/GroupByRequest';
+import { QueryBuilder } from '../Base/QueryBuilder';
+import { SimpleFilter } from './SimpleFilter';
+
+export interface ISimpleFilterOptions {
+  field: IFieldOption;
+  maximumNumberOfValues: number;
+}
+
+export class SimpleFilterValues {
+
+  static options: ISimpleFilterOptions = {
+    maximumNumberOfValues: ComponentOptions.buildNumberOption({ defaultValue: 5 }),
+    field: ComponentOptions.buildFieldOption({ required: true }),
+  };
+
+  private computedValues: string[] = [];
+  private position: number;
+
+  constructor(public simplefilter: SimpleFilter, public options: ISimpleFilterOptions) {
+    this.options.field = simplefilter.options.field;
+    this.options.maximumNumberOfValues = simplefilter.options.maximumNumberOfValues;
```

Comment:
Not needed. Options should be passed in the constructor.

See my comment about the `ComponentOptions` usage in this class.